README.rst

```diff
@@ -435,6 +426,25 @@ The recipe supports the following options:
     Allows you to specify directories and/or files which you don't want to be
     checked. Default is none.
 
+**isort**
+    If set to True, use `isort`_ to scan Python files and report about imports
+    not sorted according to the **isort-options**. Default is ``False``.
+
+**isort-exclude**
+    Allows you to specify directories and/or files which you don't want to be
+    checked. Default is none.
+
+**isort-[option]**
+    Allows you to change the way imports should be sorted. Default is to follow
+    `plone.api conventions`_. See `isort documentation for available options
+    <http://pypi.python.org/pypi/isort>`_::
+
+        ...
+        isort = True
+        isort-force-alphabetical-sort = True
+        isort-force_single_line_imports = True
+        isort-diff = True
+
```

Comment:
i just wanted to keep it consistent inside our recipe. we're using this also for flake8.